src/components/o-commercial-hero-cover/_preview.html

```diff
@@ -0,0 +1,43 @@
+<axa-commercial-hero-cover
+  src="https://i.pinimg.com/originals/b5/4c/97/b54c972b2a1138d85af5692431c02a2c.jpg"
+  gradient="black"
+  content-align="left">
+
+    <section class="js-commercial-hero-cover__first">
+      <h2 class="o-commercial-hero-cover__subtitle">Der Subtitel steht über dem Titel</h2>
```

Comment:
Please use Demo Texts in english as also AXA global can use the component.

Reply:
Done